app/views/custom_styles/show.html.erb

```diff
@@ -139,12 +156,12 @@ See docs/COPYRIGHT.rdoc for more details.
 
 <%= form_tag update_design_colors_path, method: :post, class: "form" do %>
   <section class="form--section">
-    <fieldset class="form--fieldset">
-      <legend class="form--fieldset-legend"><%= I18n.t('admin.custom_styles.custom_colors') %></legend>
+    <fieldset class="form--fieldset -collapsible collapsed">
+      <legend class="form--fieldset-legend"><%= I18n.t(:label_advanced_settings) %></legend>
 
       <% DesignColor.setables.each do |design_color|  %>
         <div class="form--field -required">
-          <label class="form--label"><%= design_color.variable %>:</label>
+          <label class="form--label"><%= I18n.t("admin.custom_styles.colors.#{design_color.variable}") %>:</label>
           <span class="form--field-container">
```

Comment:
Can we please use the chance to move the color representation right next to the input field? 

E.g. by adding
```
            <div class="form--field-affix">
              <%= icon_for_color(OpenStruct.new(variable: design_color.variable,
                                                hexcode: design_color.get_hexcode),
                                 data: { target: "#design_colors_" + design_color.variable }) %>
            </div>
```
as the first child of the `form--field-container` we get:

![image](https://user-images.githubusercontent.com/617519/72899623-af75b580-3d26-11ea-8876-7f5c56faff51.png)